subsys/shell/shell.c

```diff
@@ -1056,7 +1056,7 @@ static void shell_log_process(const struct shell *shell)
 		 * readable and can be used to enter further commands.
 		 */
 		if (shell->ctx->cmd_buff_len) {
-			k_sleep(K_MSEC(15));
+			k_sleep(15);
```

Comment:
K_TIMEOUT_MS(15) ?

Reply:
That threw me too, but `k_sleep` was not changed.  My recollection was that in discussion it was generally agreed that that API has to keep using milliseconds, just as `k_busywait` has to do seconds.  We'll need a new `k_delay()` API that uses the new capability.